webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1698,522 +1697,568 @@ msgstr "Summe der Rechnungsbeträge"
+"auszustellen."
+
+#: app/forms/steps/lotse/steuerminderungen.py:293
 msgid "form.lotse.field_handwerker_lohn_etc_summe.data_label"
-msgstr "darin enthaltene Lohnanteile, Maschinen- und Fahrtkosten, inkl. USt"
+msgstr "Betrag der Lohn-, Maschinen- und Fahrtkosten, inkl. USt"
 
-#: app/forms/steps/lotse/steuerminderungen.py:292
+#: app/forms/steps/lotse/steuerminderungen.py:298
 msgid "form.lotse.validation-haushaltsnahe-summe"
 msgstr ""
 "Sie haben angegeben, dass es haushaltsnahe Dienstleistungen gab. Bitte "
 "geben Sie hier deren Summe ein."
 
-#: app/forms/steps/lotse/steuerminderungen.py:299
+#: app/forms/steps/lotse/steuerminderungen.py:305
 msgid "form.lotse.validation-haushaltsnahe-entries"
 msgstr ""
 "Sie haben angegeben, dass es haushaltsnahe Dienstleistungen gab. Bitte "
 "geben Sie hier deren Art ein."
 
-#: app/forms/steps/lotse/steuerminderungen.py:305
+#: app/forms/steps/lotse/steuerminderungen.py:311
 msgid "form.lotse.validation-handwerker-summe"
 msgstr ""
 "Sie haben angegeben, dass es Handwerkerleistungen gab. Bitte geben Sie "
 "hier deren Summe ein."
 
-#: app/forms/steps/lotse/steuerminderungen.py:312
+#: app/forms/steps/lotse/steuerminderungen.py:318
 msgid "form.lotse.validation-handwerker-entries"
 msgstr ""
 "Sie haben angegeben, dass es Handwerkerleistungen gab. Bitte geben Sie "
 "hier deren Art ein."
 
-#: app/forms/steps/lotse/steuerminderungen.py:318
+#: app/forms/steps/lotse/steuerminderungen.py:324
 msgid "form.lotse.validation-handwerker-lohn-etc-summe"
 msgstr ""
 "Sie haben angegeben, dass es Handwerkerleistungen gab. Bitte geben Sie "
 "hier die Summe der Lohnanteile, Maschinen- und Fahrtkosten inklusive "
 "Umsatzsteuer ein."
 
-#: app/forms/steps/lotse/steuerminderungen.py:331
+#: app/forms/steps/lotse/steuerminderungen.py:337
 msgid "form.lotse.steuerminderungen.details-title"
 msgstr "Was muss ich beachten?"
 
-#: app/forms/steps/lotse/steuerminderungen.py:332
+#: app/forms/steps/lotse/steuerminderungen.py:338
 msgid "form.lotse.steuerminderungen.details-text"
 msgstr ""
 "Falls Sie in einer Mietwohnung wohnen und neben den Leistungen aus der "
-"Nebenkostenabrechnung selbständig Leistungen in Auftrag gegeben (bspw. "
-"eine Reinigungskraft) haben, können diese hier angegeben werden. Sie "
-"haben dabei die Möglichkeit mehrere Antworten auszuwählen. Bitte beachten"
-" Sie:"
+"Nebenkostenabrechnung selbständig Leistungen in Auftrag gegeben haben, "
+"können diese hier angegeben werden.</p><p><strong>Wichtige "
+"Hinweise</strong>"
 
-#: app/forms/steps/lotse/steuerminderungen.py:334
+#: app/forms/steps/lotse/steuerminderungen.py:340
 msgid "form.lotse.steuerminderungen.details-list-item-1"
 msgstr ""
-"Es <strong>muss eine Rechnung</strong> vorliegen, die zum Beispiel per "
-"Überweisung oder EC-Kartenzahlung beglichen worden ist. "
-"<strong>Barzahlungen können nicht</strong> geltend gemacht werden."
+"Es muss eine <strong>Rechnung vorliegen</strong>, die zum Beispiel per "
```

Comment:
Add "Wichtig:" in strong on the beginning

Reply:
```suggestion
"<strong>Wichtig:</strong> Es muss eine <strong>Rechnung vorliegen</strong>, die zum Beispiel per "
```

Reply:
As I see this now: Are we sure that <strong> is what we want, not <b>?
https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em